### ISSUE TITLE
Module parse failed: Unexpected token | nullish-coalescing-operator

### ISSUE THREAD

**Issue by arielwb** [reduxjs/redux-devtools#1383]
Hey all!

I've recently added this lib to the project I'm working on and I got errors both during the build of storybook and in the browser because of the nullish coalescing operator:

```
ERR! /.../node_modules/@redux-devtools/remote/lib/esm/devTools.js 213:33
ERR! Module parse failed: Unexpected token (213:33)
ERR! You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
ERR! |     } = options.filters || {};
ERR! |     this.filters = getLocalFilter({
ERR! >       actionsDenylist: denylist ?? options.actionsDenylist ?? blacklist ?? options.actionsBlacklist,
ERR! |       actionsAllowlist: allowlist ?? options.actionsAllowlist ?? whitelist ?? options.actionsWhitelist
ERR! |     });
```

transpiling `@redux-devtools/remote` fixes the issue, but I causes us to have to manage more configuration in our end specifically to use the tool.

Would you consider adding the [nullish-coalescing-operator plugin](https://babeljs.io/docs/babel-plugin-proposal-nullish-coalescing-operator) to your `babel.config.esm.json` file?

**Comment by arielwb** [comment]
Yeah, I'll try to update storybook and with that, webpack. That would partially fix the issue, since I'm also transpiling `@redux-devtools/remote` for the browser, but that is because my project uses a custom webkit version.

I understand that this issue my be specific to my context, but it would be super cool (and maybe helpful to other folks with similar restrictions) if you could release this webpack 4/older browser version.

If this is something that doesn't make sense for the lib, feel free to close this issue ;)

**Comment by clairecassan** [comment]
You can try to add a rule in your storybook main.js, for webpack:
webpackFinal: async config => {
    return {
      ...config,
      resolve: {
        ...
      module: {
        ...
        rules: [
          ...
          {
            test: /devTools.js/,
            use: {
              loader: "babel-loader",
              options: {
                presets: [["@babel/preset-env", { targets: "defaults" }]],
                plugins: ["@babel/plugin-proposal-nullish-coalescing-operator"]
              }
            }
          },
         ...
    };
